packages/orbit-components/src/InputSelect/helpers.ts

```diff
@@ -1,13 +1,17 @@
 import type { Option } from "./types";
 
+type Groups = {
```

Comment:
Same question as before, but opposite 😄 any reason for being type and not interface?

Reply:
we decided to enforce interfaces, so I will rewrite that :)